probe/kubernetes/controls.go

```diff
@@ -83,7 +83,6 @@ func (r *Reporter) CaptureResource(f func(xfer.Request, string, string, string)
 			f   func(string) (string, bool)
 		}{
 			{report.Deployment, report.ParseDeploymentNodeID},
-			{report.ReplicaSet, report.ParseReplicaSetNodeID},
```

Comment:
Now that there is just one item, we could radically simplify this entire function. Worth doing, _unless_ we believe we will be adding a new item here soon.